Smart Hunter overlay Monster missing

lucaskiu opened the pull request:
Hello, I'm here for report one missing monster no monster data.

"Em127_00" Leshen

This parts dont show up in game, smart Hunter info dont find parts 2,3,4 e etc.

"Em127_01" ancient leshen 

This monster dont show up nothing, only "unknow monster". It missing in monster data.

gabrielefilipp committed:
Initial support for Icebrone

Currently only TeamWidget (Player names and damages is working). Any help is welcome :)

gabrielefilipp committed:
Monster Widget Support

Probably won't work and status all the parts is not working currently

gabrielefilipp committed:
Fix Monster Widget

gabrielefilipp committed:
Initial Support for Monsters Parts (may crash due to empty parts list)

gabrielefilipp committed:
Something new i guess

gabrielefilipp committed:
Added an AutoUpdater

gabrielefilipp committed:
Don't know what happened to last commit (Fix AutoUpdater commit)

gabrielefilipp committed:
Fix in AutoUpdater

gabrielefilipp committed:
Fix

gabrielefilipp committed:
Found Size Address (Crown) and ofc new bugs yeee!

gabrielefilipp committed:
Added new monsters (if you find new monster ids please report it to me)

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Fixed monsters not showing in UI, fixed monster parts, added all monsters (hopefully)

gabrielefilipp committed:
Merge branch 'master' of https://github.com/gabrielefilipp/SmartHunter

gabrielefilipp committed:
Added Gold Rathian, Silver Rathalos. Fixed Glavenus typos and added more informations to README.md

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Fixed typos

gabrielefilipp committed:
Reduced start up time

For the other widgets and monster status effects I'm currently working on them

gabrielefilipp committed:
Reduced start up time

jgrrgth commented:
In SmartHunter-master\SmartHunter\bin\x64\Debug\MonsterData.json is missing this.
Just copy and paste it into MonsterData.json

    "em002_02": {
  "NameStringId": "LOC_MONSTER_SILVER_RATHALOS",
  "Parts": [
    {
      "GroupId": "Removable",
      "StringId": "LOC_REMOVABLE_PART_TAIL",
      "IsRemovable": true
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_HEAD",
      "IsRemovable": false
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_BODY",
      "IsRemovable": false
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_WING_LEFT",
      "IsRemovable": false
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_WING_RIGHT",
      "IsRemovable": false
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_LEG_LEFT",
      "IsRemovable": false
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_LEG_RIGHT",
      "IsRemovable": false
    },
    {
      "GroupId": "Part",
      "StringId": "LOC_PART_TAIL",
      "IsRemovable": false
    }
  ],
  "BaseSize": 1704.22,
  "ScaleModifier": 1,
  "Crowns": {
    "Mini": 0.9,
    "Silver": 1.15,
    "Gold": 1.23
  }
},

jgrrgth commented:
In SmartHunter-master\SmartHunter\bin\x64\Debug\MonsterData.json is missing this.
Just copy and paste it into MonsterData.json

"em001_02": {
"NameStringId": "LOC_MONSTER_GOLD_RATHIAN",
"Parts": [
{
"GroupId": "Removable",
"StringId": "LOC_REMOVABLE_PART_TAIL",
"IsRemovable": true
},
{
"GroupId": "Part",
"StringId": "LOC_PART_HEAD",
"IsRemovable": false
},
{
"GroupId": "Part",
"StringId": "LOC_PART_BODY",
"IsRemovable": false
},
{
"GroupId": "Part",
"StringId": "LOC_PART_WING_LEFT",
"IsRemovable": false
},
{
"GroupId": "Part",
"StringId": "LOC_PART_WING_RIGHT",
"IsRemovable": false
},
{
"GroupId": "Part",
"StringId": "LOC_PART_LEG_LEFT",
"IsRemovable": false
},
{
"GroupId": "Part",
"StringId": "LOC_PART_LEG_RIGHT",
"IsRemovable": false
},
{
"GroupId": "Part",
"StringId": "LOC_PART_TAIL",
"IsRemovable": false
}
],
"BaseSize": 1754.37,
"ScaleModifier": 1,
"Crowns": {
"Mini": 0.9,
"Silver": 1.15,
"Gold": 1.23
}
},

After you paste this
goto your en-US.json and replace "LOC_MONSTER_UKNOWN" with "LOC_MONSTER_GOLD_RATHIAN"

sir-wilhelm committed:
Removed unnecessary platform.

prep for #2

sir-wilhelm committed:
Set LangVersion to latest for c# 8 features/syntax.

fixes #2

sir-wilhelm committed:
Fixed gitignore for the new build directories.

sir-wilhelm committed:
Added an editor config to help VS/editors hints for code styles.

sir-wilhelm committed:
Removed unused converter.

sir-wilhelm committed:
Removed build artifacts.

I'll have to go back and either add these back and and update
"Updater.cs" to use the new paths, or re-work the auto updater.

I think it might be better to use assembly versions/tags/releases anyway

sir-wilhelm committed:
Updated SmartHunter.csproj to use the SDK format.

and set the version to 1.2.0.0

fixes #1

sir-wilhelm committed:
Code cleanup.

sir-wilhelm committed:
Added the debug compiled files back for auto updates

and updated the path to downalod the new exe.

fixes #3

sir-wilhelm committed:
Added exe to old update path so the older versions can get the auto update.

gabrielefilipp committed:
Added statuses (NOT rage & stamina), and bug fixes

sir-wilhelm committed:
Merge branch 'dev' into code-cleanup

# Conflicts:
#	SmartHunter/bin/x64/Debug/SmartHunter.exe

gabrielefilipp committed:
Merge pull request #4 from sir-wilhelm/code-cleanup

Code cleanup

gabrielefilipp committed:
Code cleanup

gabrielefilipp committed:
Fixed AutoUpdater

gabrielefilipp committed:
Added rage timer

gabrielefilipp committed:
Bug fix

gabrielefilipp committed:
Bug fix

tvang1 commented:
Hi, so I've been adding monsters into the MonsterData.json and the MonsterDataConfig.cs. I've run into a couple of problems with Shara-Ishvalda. The parts list fills the screen and it makes gameplay difficult. Is there a way to make parts disappear at different stages?

For the guiding lands player widget, is the solution to make a new widget for the guiding lands? I believe there is a new pattern for players there but I can't find it. Having a new widget for the guiding lands may be one solution to fix the pattern there.

gabrielefilipp committed:
Fixed array overflow (hopefully)

tvang1 commented:
There is still an array overflow. [https://imgur.com/a/2H1DIb9](url)

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Bug fix?

gabrielefilipp committed:
Added new statuses

gabrielefilipp committed:
Fixef monster sizes

gabrielefilipp committed:
Bug fix

gabrielefilipp committed:
Added Monsters Removable Parts

gabrielefilipp committed:
Create LICENSE.txt

zakuciael committed:
AutoUpdater will now replace the default exe instead of appending hash to the executable.

zakuciael committed:
Code cleanup (Optimized imports)

gabrielefilipp committed:
Fix for the new patch

zakuciael committed:
Merge branch 'master' of https://github.com/r00telement/SmartHunter

zakuciael committed:
Updated the executable after merging upstream.

zakuciael committed:
Merge branch 'master' of https://github.com/gabrielefilipp/SmartHunter

gabrielefilipp committed:
Merge pull request #21 from zakuciael/master

AutoUpdater changes + code cleanup

gabrielefilipp committed:
Fixed crash

gabrielefilipp committed:
Initial support for player widget

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
New stuff (and bugs)

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Bug fix

gabrielefilipp committed:
Buf fix for new patch

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Add files via upload

Give me your thoughts and if I should push or not this update

gabrielefilipp committed:
Monster data sharing with server

gabrielefilipp committed:
Bug fix

gabrielefilipp committed:
Added expedition damage calculation

Please read carefully the new .PDF document that I have uploaded

gabrielefilipp committed:
Fixed server

gabrielefilipp committed:
Crash fix

gabrielefilipp committed:
Added new monsters localization

gabrielefilipp committed:
Added settings tab

Huge thank to Phil-Pa

gabrielefilipp committed:
Fixed monster data

AxeBane committed:
Fix typo

dragonyue0417 committed:
Fix Buff, Add Monster

dragonyue0417 committed:
Update Test

dragonyue0417 committed:
2020.04.13.01

dragonyue0417 committed:
Fix Anger not displayed

Fix Anger not displayed
修正魔物憤怒不顯示

dragonyue0417 committed:
Fix buff

Memory PatternString need fix
新增溫泉、冰耐草、鬼毬果Buff

dragonyue0417 committed:
Fix Buff

修正與新增Buff

dragonyue0417 committed:
2020.04.17.00

2020.04.17.00

dragonyue0417 committed:
Fix Wording

dragonyue0417 committed:
Copy damage and percentage to clipboard.

Add hotkey to copy Team or Player damage percentage to clipboard.

gabrielefilipp committed:
Merge pull request #74 from dragonyue0417/master

 Fix Buff, Add Monster

gabrielefilipp committed:
Merge pull request #72 from AxeBane/patch-1

Fix typo

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Update README.md

gabrielefilipp committed:
Create smarthunter.php
